inc/common/admin-bar.php

```diff
@@ -32,221 +28,70 @@ function rocket_admin_bar( $wp_admin_bar ) {
 		[
 			'id'    => 'wp-rocket',
 			'title' => WP_ROCKET_PLUGIN_NAME,
-			'href'  => admin_url( 'options-general.php?page=' . WP_ROCKET_PLUGIN_SLUG ),
-		]
-	);
-
-	/**
-	 * Settings.
-	 */
-	$wp_admin_bar->add_menu(
-		[
-			'parent' => 'wp-rocket',
-			'id'     => 'rocket-settings',
-			'title'  => __( 'Settings', 'rocket' ),
-			'href'   => admin_url( 'options-general.php?page=' . WP_ROCKET_PLUGIN_SLUG ),
+			'href'  => current_user_can( 'rocket_manage_options' ) ? admin_url( 'options-general.php?page=' . WP_ROCKET_PLUGIN_SLUG ) : '#',
```

Comment:
I think that using `false` or an empty string instead of `'#'` will display a `<div>` instead or a link.
https://github.com/WordPress/WordPress/blob/master/wp-includes/class-wp-admin-bar.php#L546-L549